Combobox.js

```diff
@@ -684,7 +706,7 @@ define([
 				// save what user typed at each keystroke.
 				this.value = inputElement.value;
 				if (this._useCenteredDropDown()) {
-					this.inputNode.value = inputElement.value;
+					this.displayedValue = inputElement.value;
```

Comment:
I filed #688 for the remaining task to keep displayedValue in sync w/the actual displayed value.